Client/Frontend/Browser/SearchViewController.swift

```diff
@@ -13,10 +13,15 @@ private enum SearchListSection: Int {
     case Search
 }
 
+protocol SearchViewControllerDelegate: class {
```

Comment:
All of these table views we have are ways to show lists of urls or sites. I guess I don't see them as attached to views, but if you wanted to view it that way, they're all attached to SiteTableViewControllers. We did that intentionally because they all support shared display types and shared functionality.

These ViewControllers shouldn't care where they're displayed, if its on Home or in some other place. They're not really HomePanels. HomePanels are (IMO) just a convenience for generating these ViewControllers and maintaining a list of which ones to show on about:home.